File: html/old/R2/SACO R2 2007 (Solutions)/triangle.cpp
/*
 * Solution to question "Triangle" from the
 * 2007 South African Computer Olympiad
 *
 * Carl Hultquist (chultquist@gmail.com)
 */

#include <iostream>

using namespace std;

int main()
{
    int a, b, c;
    cin >> a >> b >> c;
     if ((a == b) && (b == c))
        cout << "EQUILATERAL" << endl;
    else if ((a == b) || (b == c) || (c == a))
         cout << "ISOSCELES" << endl;
    else
         cout << "SCALENE" << endl;
    return 0;
}

File: html/old/R2/SACO R2 2007 (Solutions)/triangle.pas
{
  Solution for 'Triangles'
  SACO Problem 2
  2nd round 2007
  By Ben Steenhuisen
}
program triangle;
uses crt;
var x,y,z:integer;
begin
   clrscr;
   read(x);read(y);read(z);
   if ((x=y) and (x=z)) then
   begin
      writeln('EQUILATERAL');
   end
   else
   begin
      if ((x=y) or (x=z) or (y=z)) then writeln('ISOSCELES')
      else writeln('SCALENE');
   end;
end.
